src/org/labkey/test/tests/wiki/WikiTest.java

```diff
@@ -149,6 +152,25 @@ public void testSteps()
         assertTextPresent("This folder does not currently contain any wiki pages to display");
     }
 
+    @Test
+    public void testShowPageTreeForWiki()
+    {
+        goToProjectHome();
+        log("Creating the wiki");
+        WikiHelper wikiHelper = new WikiHelper(this);
+        wikiHelper.createNewWikiPage("HTML");
+        setFormElement(Locator.name("name"), WIKI_PAGE_NAME);
+        setFormElement(Locator.name("title"), WIKI_PAGE_TITLE);
+        wikiHelper.setWikiBody(WIKI_CHECK_CONTENT);
+        wikiHelper.saveWikiPage();
+
+        log("Verifying the tree is displayed correctly");
+        goToProjectHome();
+        EditPage editWikiPage = wikiHelper.editWikiPage(WIKI_PAGE_NAME);
+        editWikiPage.clickShowPageTree();
+        assertElementPresent(Locator.linkContainingText(WIKI_PAGE_TITLE + " (" + WIKI_PAGE_NAME + ")"));
```

Comment:
Not strictly necessary, but it would be nice to verify that the link is actually inside the wiki tree.
```suggestion
        assertElementPresent(Locator.id("wiki-toc-tree").append(Locator.linkContainingText(WIKI_PAGE_TITLE + " (" + WIKI_PAGE_NAME + ")")));
```